databases/week2/creatingDatabase.sql

```diff
@@ -0,0 +1,33 @@
+create database carRepair;
+use carRepair;
+-- drop database carRepair;
+
+CREATE TABLE `cars` (
+  `license_no` varchar(50) NOT NULL,
+  `owner` varchar(255) NOT NULL,
+  `model` varchar(25) not null,
+  
+  PRIMARY KEY (`license_no`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `services` (
+  `service_no` int(10) NOT NULL,
+  `type` varchar(50) NOT NULL,
+  `price_per_hour` int(3) not null default 600,
+  
+  PRIMARY KEY (`service_no`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `repairs` (
+  `license_no` varchar(50) NOT NULL,
```

Comment:
define a primary key column or unique column  for this table

Reply:
I addeded repairs_no as a primary key.